app/views/admin/reports/good_jobs/export.csv.erb

```diff
@@ -0,0 +1,14 @@
+<%- headers = ['queue_name', 'job_class', 'job_id', 'performed_at', 'finished_at', 'time_to_completion'] -%>
+<%= CSV.generate_line headers %>
+<%- @jobs.each do |job| -%>
+<%= CSV.generate_line(
+  [
+    job['queue_name'],
+    job['job_class'],
+    job['job_id'],
+    job['performed_at'],
+    job['finished_at'],
+    job['time_to_completion_in_seconds']
```

Comment:
Assume this is not supposed to be time_to_completion as per the header?

Reply:
Otherwise I would suggest using job.values_at(*headers) here. Sorry if I misunderstood

Reply:
:+1: